stdlib/public/core/Mirror.swift

```diff
@@ -660,8 +660,22 @@ internal extension Mirror {
 
 //===--- QuickLooks -------------------------------------------------------===//
 
-/// The sum of types that can be used as a Quick Look representation.
+/// `PlaygroundQuickLook` is deprecated, and will be removed from the standard
+/// library in a future Swift release. Customizing display for in a playground
+/// is now done using the `CustomPlaygroundDisplayConvertible` protocol, which
+/// does not use the `PlaygroundQuickLook` enum. Please remove your uses of
+/// `PlaygroundQuickLook`, or conditionalize your use such that it is only
+/// present when compiling with Swift 4.0 or Swift 3.2 or earlier:
+///
+/// ```swift
+/// #if !(swift(>=4.1) || swift(>=3.3) && !swift(>=4.0))
```

Comment:
Our Markdown style for examples is to use indented code blocks, not fenced ones, unless we're writing non-Swift code. The code itself should be indented with 4 spaces.